cornflow-server/cornflow/tests/custom_test_case.py

```diff
@@ -227,13 +233,13 @@ def get_no_rows(self, url, token=None):
         return rows.json
 
     def update_row(
-        self,
-        url,
-        change,
-        payload_to_check,
-        expected_status=200,
-        check_payload=True,
-        token=None,
+            self,
+            url,
+            change,
+            payload_to_check,
+            expected_status=200,
+            check_payload=True,
+            token=None,
```

Comment:
Same to this formatting change. If I run black locally it changes back this changes, which version of black are you running?